CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Breaking changes
 
 - Geodesic interpolation is now used in `Flight.resample_and_fill` when the great circle distance between waypoints (rather than the total segment length including vertical displacement) exceeds a threshold. This may change the interpolation method used when resampling flight segments with lengths close to the geodesic interpolation threshold.
+- Bugfix will decrease computed values of moist heat capacity with non-zero specific humidity.
```

Comment:
```suggestion
- Fixed typo in `thermo.c_pm`  will decrease computed values of moist heat capacity with non-zero specific humidity. This update results in a negligible (<<1%) change in contrail predictions by `ISSR`, `SAC`, `PCR, and `Cocip` models.
```

Reply:
I'm a little hesitant to put a number on the downstream changes without actually quantifying them. The changes to values in pinned fixtures are all tiny, though, so I agree that the changes are likely to be negligible. How about
```
Fixed typo in `thermo.c_pm`  will decrease computed values of moist heat capacity with non-zero specific humidity. We expect the downstream impact on contrail predictions by `ISSR`, `SAC`, `PCR, and `Cocip` models to be negligible.
```
instead?

Reply:
I agree - and probably best to characterize as `minimal` rather than `negligble`. Thanks!

Reply:
https://github.com/contrailcirrus/pycontrails/pull/185/commits/d21451bc7c60f32b9e717cb22ebc1de455254ba3